docs/source/app-dev/bindings-java/codegen.rst

```diff
@@ -141,6 +143,19 @@ Java:
 
 .. _com.daml.ledger.javaapi.data.Unit: https://docs.daml.com/app-dev/bindings-java/javadocs/com/daml/ledger/javaapi/data/Unit.html
 
+Understand escaping rules
+-------------------------
+
+To avoid clashes with Java keywords, the Java codegen applies escaping rules to the following DAML identifiers:
+
+#. Type names (except the already mapped :ref:`built-in types <daml-codegen-java-primitive-types>`)
```

Comment:
I don't think you need a numbered list for this, unless it is ordered and I hadn't realised?

Reply:
Oh, good point. There's definitely no order to it, and I also don't refer to any specific point later. 👍

Reply:
Fixed in https://github.com/digital-asset/daml/pull/281/commits/4d848c719a057b2337f492901fe1a68cd76ca35d.